electron/collectionRenderer.js

```diff
@@ -40,6 +40,16 @@ let sortBoosters = (boosters) => {
   })
 }
 
+let padBoosters = (boosters) => {
+  let collationIds = [100005,100006,100007,100008,100009,100010]
+  for (collationId of collationIds) {
+    if (!boosters.find((x) => { return x.collationId == collationId})) {
```

Comment:
Took me a minute to figure out this is doing:
`!boosters.find(x => x.collationId == collationId)`

Reply:
I am pretty sure the answer to this is no, but is there some way we can use like... js closures or something to share `collationIds` with the webworker? (I think I tried to do something like this with the cardsets and it didn't work, so maybe it's a big shrug)